### ISSUE TITLE
continually prompted to select C# startup project

### ISSUE THREAD

**Issue by burtonrodman** [microsoft/vscode-dotnettools#1440]
Type: <b>Bug</b>

1. open a workspace that has about 8 repos referenced, each with a solution including many C# projects
2. try to build... prompted for C# startup project
3. try to debug... prompted for C# startup project
4. Reload Window... prompted for C# startup project
5. try to debug... prompted for C# startup project TWICE

Extension version: 1.11.4
VS Code version: Code 1.93.0 (4849ca9bdf9666755eb463db297b69e5385090e3, 2024-09-04T13:02:38.431Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|13th Gen Intel(R) Core(TM) i9-13900H (20 x 2995)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.59GB (9.78GB free)|
|Process Argv|--file-uri file:///d%3A/source/2010solutions/VisitingAngels/VisitingAngels.Aspire/VisitingAngels.Aspire.code-workspace --crash-reporter-id c8e3650d-a133-42e9-901c-3aa8e8465629|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256860
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
jg8ic977:31013176
a69g1124:31058053
dvdeprecation:31068756
dwnewjupyter:31046869
impr_priority:31102340
refactort:31108082
pythonrstrctxt:31112756
wkspc-onlycs-t:31132770
wkspc-ranged-c:31125598
ei213698:31121563

```

</details>

**Comment by burtonrodman** [comment]
this is on top of tracing down a MethodNotFoundException that wasted half the day (my fault); lots of false positive, random C# errors (mostly razor files), ... basically zero productivity.

I'm sure this is just an issue in a preview version, but I know if i go back to the Release version I get to deal with all the build and notification error there.

I'm sorry, but it sucks to be a C# developer in vscode right now 😠

**Comment by tmeschter** [comment]
@burtonrodman Do you have this setting turned on?

![image](https://github.com/user-attachments/assets/aca5c68a-57e8-4b89-83f6-cbaec036eb34)

**Comment by burtonrodman** [comment]
yes, that setting was on.

**Comment by smitpatel** [comment]
Most likely duplicate of #1464

**Comment by smitpatel** [comment]
Moving to area-maui. Mostly fix for #1464 should fix this. But if appears after updated MAUI extension, we can investigate.

**Comment by emaf** [comment]
As @smitpatel mentioned, this is a duplicate of #1464.
